src/main/java/com/easypost/net/EasyPostResource.java

```diff
@@ -740,11 +692,51 @@ public String getStatus() {
         return "";
     }
 
+    // Tracker
+
     /**
      * @return the tracking details of this shipment
      */
     public List<TrackingDetail> getTrackingDetails() {
         return new ArrayList<TrackingDetail>();
     }
 
+    protected enum RequestMethod {
+        GET, POST, DELETE, PUT
```

Comment:
Comma separate lines read better one item per line and sorted, can we do that here?

Reply:
We can, if I can find the .editorconfig setting to override enum values formatting

Reply:
Ok, don't spin wheels on it if it's not immediately obvious. Generally though that's the idea. One odd list won't break the world.

Reply:
Resolved in 28156240aa91125dfeeb163cd5eb6d71869b762a